dave/template.md

```diff
@@ -0,0 +1,68 @@
+# My Corp Inc.
```

Comment:
we don't need dave. that was just a demo i forgot to .gitignore. will remove those files, camelCase everything, and take a look at that loop.